include/sys/ringbuf.h

```diff
@@ -26,6 +26,7 @@ bool ringbuf_putb(ringbuf_t *buf, uint8_t byte);
 /*! \brief Put exactly n bytes into buf if there's enough space. */
 bool ringbuf_putnb(ringbuf_t *buf, uint8_t *data, size_t n);
 bool ringbuf_getb(ringbuf_t *buf, uint8_t *byte_p);
+bool ringbuf_getnb(ringbuf_t *buf, uint8_t *data, size_t n);
```

Comment:
I would provide a comment similar to `ringbuf_putnb`

Reply:
Done.